processing/data_collection/gazette/spiders/sp_marilia.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpMariliaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3529005"
+    name = "sp_marilia"
+    allowed_domains = ["marilia.sp.gov.br"]
+    start_urls = ["https://www.marilia.sp.gov.br/portal/diario-oficial"]
+
+    def parse(self, response):
+        gazettes = response.css(".d_e_modelo_diario")
+        for gazette in gazettes:
+            gazette_url = gazette.css("a::attr(href)").getall()[1]
+            is_extra_edition = gazette.css(".d_e_btextra::text")[1:]
+            if gazette_url:
+                gazette_date = parse(
+                    gazette.css(".d_e_cont_bt span::text").get()[1:11], languages=["pt"]
+                ).date()
+                file_urls = response.urljoin(gazette.css("a::attr(href)").getall()[1])
```

Comment:
You already got `gazette_url`. There is no need to do it again. Just add the `response.urljoin` and everything is fine then.